src/java/org/apache/cassandra/tcm/serialization/Version.java

```diff
@@ -38,6 +38,10 @@ public enum Version
      *  - Serialize MemtableParams when serializing TableParams
      */
     V2(2),
+    /**
+     *  - Serialize allowAutoSnapshot and incrementalBackups when serializing TableParams
+     */
+    V3(3),
```

Comment:
think you need to update `org.apache.cassandra.tcm.membership.NodeVersion#CURRENT_METADATA_VERSION`?